contracts/src/v0.8/automation/dev/v2_3/AutomationRegistry2_3.sol

```diff
@@ -229,10 +233,12 @@ contract AutomationRegistry2_3 is AutomationRegistryBase2_3, OCR2Abstract, Chain
    * @param amount number of LINK transfer
    */
   function onTokenTransfer(address sender, uint256 amount, bytes calldata data) external override {
```

Comment:
I think this is called when we do transferAndCall from Registrar? If so, the billing token could be non-Link, isn't it?

Reply:
nope, only LINK can call this function :) the registrar will call `addFunds()` when the billing token is non-link (I just made this change after your previous call out)